### ISSUE TITLE
Steam friends is showing im playing multiple instances of 

### ISSUE THREAD

**Issue by shotzpk** [uberhalit/EldenRingFpsUnlockAndMore#49]

**Comment by shotzpk** [comment]
So my friends are telling me there's heaps of accounts with my name on steam friends list showing i'm play Elden ring

**Comment by DubbleClick** [comment]
Is it only happening when you launch through the tool? What if you launch eldenring.exe manually and then patch it with the tool?

**Comment by uberhalit** [comment]
Are you running the game in online mode? I have a very hard time comprehending this, the utility simply starts steam if it isn't running already. Steam does not get modified or altered it any way.

**Comment by shotzpk** [comment]
> Are you running the game in online mode? I have a very hard time comprehending this, the utility simply starts steam if it isn't running already. Steam does not get modified or altered it any way.

I'm not even running it through the tool. I ran it through the tool in offline mode. But i didnt close steam when i did. And now when i launch the game without the mod. It does this.. Cause i play online with my mates sometimes.

**Comment by DubbleClick** [comment]
that doesn't sound related to this tool at all. it only does game memory (ram) manipulation and has no effects that last through a restart.

**Comment by shotzpk** [comment]
> that doesn't sound related to this tool at all. it only does game memory (ram) manipulation and has no effects that last through a restart.

Its just weird that its only done this after I got the tool. :/ No one else I see has multiple instances of the game on my friends list or anyone elses.

**Comment by shotzpk** [comment]
People keep messaging me about it lol

**Comment by shotzpk** [comment]
I feel like im gonna get banned and my money is gonna go down the drain..

**Comment by el0c** [comment]
This happens with steam games semi-often, just quit and restart steam. It's fine. Steam just failed to realize the process closed and says it's still running.

**Comment by shotzpk** [comment]
> This happens with steam games semi-often, just quit and restart steam. It's fine. Steam just failed to realize the process closed and says it's still running.

Its weird tho, Too weird for my liking. Like normally I see my friends games launch multiple times, but nothing like this.. where I would appear multiples times on sometimes friends list.

**Comment by shotzpk** [comment]
Btw... its not saying im still playing the game when its closed. Its not saying bottom right in the pop up launching heaps of times. This is actual friends list showing like 20 of me playing elden ring.. sorry i was tired this morning when i typed this

**Comment by el0c** [comment]
Can you get a screenshot from your friends of this then?

**Comment by shotzpk** [comment]
Yeah, i do have a screenshot of it from one of my friends. Who do i send it to? I dont wanna send it here.

**Comment by el0c** [comment]
Why? I don't see a reason why it needs to remain private. There's very little if any extra identifying information in a Steam Friends list that wouldn't be available on a Steam profile, even if it was set to private as far as I can remember.

**Comment by shotzpk** [comment]
Yeah, i'll do that.

Here it is.. All the profile pictures that is an orange face is all me... 
There's also 10 more under the next friend on my friends friend list..
[Elden ring repeated friend image](https://imgur.com/ILfl7m1)

**Comment by el0c** [comment]
Very interesting. So that's the "Friends who play section" when they're looking at the game in their library. 
Not sure what would cause that at all. Having looked at the source for the tool I definitely don't see any modifying of game files. Probably just coincidence and steam doing what steam does and bugging out sometimes in the weirdest of ways, and that itself isn't going to get you banned, in my opinion. So I wouldn't worry to much about it.

Is it persistent, (happening every time you run the game without the Unlock tool?)

**Comment by shotzpk** [comment]
> Very interesting. So that's the "Friends who play section" when they're looking at the game in their library. Not sure what would cause that at all. Having looked at the source for the tool I definitely don't see any modifying of game files. Probably just coincidence and steam doing what steam does and bugging out sometimes in the weirdest of ways, and that itself isn't going to get you banned, in my opinion. So I wouldn't worry to much about it.
> 
> Is it persistent, (happening every time you run the game without the Unlock tool?)

Thanks for replying.
It seemed to happen for a few days, today i asked my mate if it was doing it and he said it wasnt. Hopefully it doesnt get me banned or anything and hopefully its just steam bugging out in the weirdest ways.

**Comment by uberhalit** [comment]
> Yeah, i'll do that.
> 
> Here it is.. All the profile pictures that is an orange face is all me... There's also 10 more under the next friend on my friends friend list.. [Elden ring repeated friend image](https://imgur.com/ILfl7m1)

This is definitely steam bugging out. I don't see how the patcher could possibly cause that, especially because the game is in offline mode.
